### ISSUE TITLE
PluginClassLoader fails to load compiled class

### ISSUE THREAD

**Issue by jvtrigueros** [JetBrains/gradle-intellij-plugin#79]
I'm consistently getting a `ClassNotFoundException` every other time I run `runIdea`. Here's a [stacktrace](https://gist.github.com/jvtrigueros/a164835fcd5d85c6174d) of what happens when the exception is thrown. I'd love to provide better information but I don't know how. Here's the [repo ](https://github.com/guacamoledragon/twitch-chat) that I'm building. Could it be an issue with the Kotlin plugin?

**Comment by zolotov** [comment]
cannot recreate. try to remove `ls ~/.gradle/caches/` directory and try again.

**Comment by jvtrigueros** [comment]
I tried this on an Ubuntu VM and on a Macbook Pro. I wasn't able to repro it on the Ubuntu VM but it manifested itself on OS X.

However, I believe I've found part of the issue, if the file `.gradle/<gradle-version>/taskArtifacts/cache.properties.lock` is deleted before running `./gradlew runIdea`, the above exception will no longer happen.

I'm sure this is probably a symptom to the real issue, but for now I've added the following to my `gradlew` and `gradlew.bat`

**grablew**
```sh
rm .gradle/2.5/taskArtifacts/cache.properties.lock
```

**gradlew.bat**
```batch
del %~dp0\.gradle\2.5\taskArtifacts\cache.properties.lock
```
